cli/src/etos_client/downloader/downloader.py

```diff
@@ -142,6 +107,49 @@ def __save_file(self, item: Downloadable, response: requests.Response) -> None:
         with open(download_path, "wb+") as report:
             for chunk in response:
                 report.write(chunk)
+        if not self.__verify(item, download_path):
+            self.logger.error("Checksum verification failed on %r", item)
+
+    def __verify(self, item: Downloadable, path: Path) -> bool:
+        """Verify the checksum of the downloaded item.
+
+        Path is the real path to the saved file.
+        """
+        translation_table = {
+            "SHA-224": "sha224",
+            "SHA-256": "sha256",
+            "SHA-384": "sha384",
+            "SHA-512": "sha512",
+            "SHA-512/224": "sha512_224",
+            "SHA-512/256": "sha512_256"
+        }
+        hash = None
+        expected_digest = None
+        for nist_scheme, python_scheme in translation_table.items():
+            if item.checksums.get(nist_scheme) is not None:
+                hash = hashlib.new(python_scheme)
```

Comment:
in sse.rst the examples show checksums as empty strings. Should this code also check if the checksum isn't an empty string?